### ISSUE TITLE
The app doesn't show image of icon properly in the notifications

### ISSUE THREAD

**Issue by Kushalrock** [Routinger/routinger#5]
**Describe the bug**
The app doesn't show the icon of the app in the notifications. 

**To Reproduce**
1. Go to the floating action button
2. Add any scheduled or recurring notification.
3. Wait for the notification to arrive
4. You will see a white block in place of the icon. [SEE IMAGES BELOW]

**Expected behavior**
The app's icon should have been shown in place of a white square block.

**Screenshots**
![Screenshot_20210925-012352](https://user-images.githubusercontent.com/64556649/134765874-ff5395db-6746-4d0d-aec9-18267e5ffeb6.jpg)

**Additional context**
The icon that should be used in the notifications is initialized in `lib/services/notifications.dart`.

**Ways to solve it**
Changing the icon size or creating proper adaptive icons.

**Ideas about why is this happening**
I am not completely sure, but perhaps the issue is the use of non-proper adaptive icons and the size of the icon.

**Comment by AyoubBoulahtar** [comment]
Hey, can you assign me to this issue ?

**Comment by Kushalrock** [comment]
Yes I can assign you to the issue. 

What do you think the problem is?? Is it the same that I have mentioned??

**Comment by AyoubBoulahtar** [comment]
The notifications icon works fine for my device, so its definitely the adaptive constraints.

**Comment by Kushalrock** [comment]
Okay so how are you going to make the adaptive icons??

**Comment by Kushalrock** [comment]
In my real device the app icon also doesn't look that good it should show the whole clock
![Screenshot_20210927-144741.jpg](https://user-images.githubusercontent.com/64556649/134880826-54ba5f62-d7fd-4156-af0f-9a1895f2477f.jpg)

**Comment by msramesh2k1** [comment]
Hey @Kushalrock can I take this issue , still it is not resolved

**Comment by msramesh2k1** [comment]
> Also @msramesh2k1 , I checked out your github profile and really intrigued by it. We have a major bug and we are working on it. If you can help us with that too. The bug is with recurring notifs we can schedule them but I guess that either due to flutter local notif package that we are using for notifications, it works some time but other times when app is in background some notifs doesn't show up. If it is a problem from our side we will try our best to resolve it, but if its a problem from flutter local notif package side I guess we have to shift to using android alarm manager.

Yes @Kushalrock it won't work perfectly in background or once app is closed  , I too tired lots of package such as cron , android alarm manager but finally I end up with background fetch . You can also give a try on background_fetch

**Comment by Kushalrock** [comment]
I see background fetch maybe you can work with us shifting to it. @msramesh2k1

**Comment by Kushalrock** [comment]
Sure, No problem. Thank you for helping

**Comment by alamin-karno** [comment]
> Actually, I don't know if it actually works or not. Because we have some phones where it works like on my real device and one more device, and others where it doesn't show image like in Android emulator and one more device I tested on. You can check it. And tell me if changes are needed.

It can be solved by using the adaptive icon. This is right, but with an adaptive icon, you can only solve this. You also have to use that adaptive icon for Android devices too. Also, if you want, you can add custom colours from the Android Manifest file. I hope I can help.

**Comment by Kushalrock** [comment]
@alamin-karno I can assign you this task

**Comment by Priyammm05** [comment]
Is the issue solved ?
